hasher-matcher-actioner/.devcontainer/.gitignore

```diff
@@ -0,0 +1,2 @@
+# You likely have to customize this, so copy it
+# devcontainer.json
```

Comment:
It was unclear to me what the behavior would be if I checked this in the same commit as moving it, so do it in two steps.

Reply:
.gitignore is 'observed' at `git add`, so same commit would have worked.